vms/evm/imports_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"fmt"
+	"go/parser"
+	"go/token"
+	"io/fs"
+	"path/filepath"
+	"slices"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const root = ".."
```

Comment:
Declarations should be made close to where they're used, not arbitrarily at the top of a file. The "rule" of placing constants at the top of a file is not a thing in Go.